rhel/Dockerfile

```diff
@@ -20,7 +18,7 @@ ARG D_WITHOUT_FLAGS="--without-rshim-dkms --without-iser-dkms --without-isert-dk
 WORKDIR /root
 ADD ${D_OFED_URL_PATH} ./
 RUN tar -xf ${D_OFED_TARBALL_NAME}
-RUN dnf -y install make gcc wget perl createrepo kernel-core-$(uname -r) kernel-devel-$(uname -r) pciutils python36-devel ethtool lsof elfutils-libelf-devel rpm-build kernel-rpm-macros python36 tk numactl-libs libmnl tcl binutils kmod procps git autoconf automake libtool
+RUN dnf -y --enablerepo=rhel-8-for-x86_64-baseos-eus-rpms --releasever=${D_OS_VERSION} install make gcc wget perl createrepo kernel-core-$(uname -r) kernel-devel-$(uname -r) pciutils python36-devel ethtool lsof elfutils-libelf-devel rpm-build kernel-rpm-macros python36 tk numactl-libs libmnl tcl binutils kmod procps git autoconf automake libtool hostname
```

Comment:
This repo is not available in registry.access.redhat.com/ubi8:latest image.

Reply:
Indeed, it is not.. You need the entitlements on the RHEL host where you are building it.

Reply:
We need also need to enable `rhocp-${OPENSHIFT_VERSION}-for-rhel-8-x86_64-rpms` see the script in the NV GPU operator: https://gitlab.com/nvidia/container-images/driver/-/blob/master/rhel8/nvidia-driver#L54

Reply:
EUS is not provided for odd minor releases, 8.3, 8.5, etc.
We need to update the instruction to be able to build images for all releases.